portfolio/src/main/webapp/script.js

```diff
@@ -158,33 +158,37 @@ async function checkLogin(){
 
 /* Creates a map and adds it to the page. */
 function initMap() {
-  const jakarta = {lat: -6.175540, lng: 106.82743};
-
-  const map = new google.maps.Map( document.getElementById("map"), {
-    center: jakarta,
-    zoom: 15,
-    mapTypeControlOptions: { mapTypeIds: ['roadmap', 'satellite', 'hybrid', 'terrain', 'night_map'] }
+  fetch("/city-data").then(result => result.json()).then((cities) => {
+      console.log(cities);
+      const map = new google.maps.Map( document.getElementById("map"), {
+        center: {lat: -6.175540, lng: 106.82743},
```

Comment:
nit: please add an inline comment or leave it as a named constant. just to know what are these numbers for :)